docs/_tec-events/2023-11-06-swcsf-meeting.md

```diff
@@ -0,0 +1,74 @@
+
+Section 702, itself part of the FISA Amendments Act of 2008, enables federal officials to conduct surveillance on anyone outside of the U.S. in the interest of national security.
+
+The problem is the law was written in a way that has enabled egregious abuses, including the mass surveillance of Americans on U.S. soil, the vast majority of whom have nothing to do with terrorist plots or nefarious acts.
+
+The FBI and the Biden Administration says Section 702 essential for protecting national security. But legal scholars, government watchdog groups and internet privacy advocates say Section 702 has been widely abused and does little, if anything, to protect Americans from threats.
+
+## Presentation 2: Breakdown in Communication
+
+Breakdown in Communication: Modern Communication…and why we don’t have it.
+
+Elio Grieco will discuss the state of modern, digital communications, popular and obsucre messaging programs, and what’s missing from all of them.
+
+What are the basic building blocks of a truly modern communication system? Identity, data sync, and data curation and discovery.
+
+We’ll take a look at past, current, and upcoming digital communication programs like IRC, Signal, SimpleX and others. What are their features, and what features are missing? What trends do we see in feature set, and how has feature set influenced modern communications and general human cognition?
+
+What are the pressures leading us into such app designs? What are the threats to communication? Well talk about the Crypto Wars and the 4 Horesmen of the Infocalypse, including the current threats coming out of Europe and the US.
+
+We’ll end with a list of organizations that are championing important communication research, giving everyone the info they need to follow these trends and get involved in correcting them
+
+## Location
+
+For this meeting, we are back in Room SO1330 of the South Building at GateWay Community College, Washington Campus. Details below.
+Schedule
+
+### Attend in Person
+
+Room SO1330 of the South Building at GateWay Community College, Washington Campus.
+
+108 N 40th St,
+Phoenix, AZ 85034
+
+Map link to the [Exact Location](https://www.google.com/maps/place/South+Building/@33.4484729,-111.9984009,19z/data=!3m1!4b1!4m6!3m5!1s0x872b0e87ca3e5fc9:0xf58ab1600bd355fc!8m2!3d33.4484729!4d-111.9977572!16s%2Fg%2F1hdzm1g1l?shorturl=1)
+
+#### Parking Map
+Map of campus and parking options at Gateway Community College with SWCSF and the ACSL highlighted.
+![Parking Map](/assets/images/events/gateway_map.png)
+
+#### South Building Map
+Floorplan with room SO1330 highlighted for the South Building at Gateway Community College
+![Building Map](/assets/images/events/swcsf-south-building-map.png)
+
+## Food
+
+Free pizza and soft drinks will be provided by TEC. If you have other preferences, Speedy Street Tacos is within walking distance, just past the East parking lot. There are also a few other restaurants within a short drive of the college.
```

Comment:
This seems to be repeated across events, consider making this into an `include`.

https://jekyllrb.com/docs/includes/

Reply:
Future PR